#164198481 setup sequelize ORM

danprocoder opened the pull request:
#### What does this PR do?
Sets up sequelize ORM for the project.

#### Description of Task to be completed?
1. Uninstall mongoose modules.
2. Install sequelize and sequelize-cli modules and setup sequelize ORM to the postgres database for the project.
3. Creates PR template.

#### What are the relevant Pivotal Tracker stories?
[#164198481](https://pivotaltracker.com/story/show/164198481)

danprocoder committed:
chore(sequelize): Initialized sequelize for the project
1. Uninstalled mongoose modules
2. Initialized sequelize. Database related files (models, migrations, seeders and config) are saved in /database relative to the root path.
[Finishes #164198481]

danprocoder committed:
chore(databaseConfig): Updated database config.json file
Configured sequelize to use DATABASE_URL from .env file for database connections for development, test and production
[Finishes #164198481]

danprocoder committed:
chore(prTemplate): created template file for pull requests
Created pull_request_template.md file
[Finishes #164198481]

andela-gike reviewed (changes requested):
The PR looks good to me, but could you configure babel so we can start using imports
You also need to update your package.json file with the script to run creating a DB, running all the necessary migration for the application etc.

danprocoder committed:
chore(sequelize): Updated database configuration
- Installed pg module
- Changed config.json to config.js so as to load environment variables programmatically before configurations are exported as sequelize is configured to use DATABASE_URL environment variable for database connections.
[Finishes #164198481]

danprocoder committed:
chore(babel): Installed babel
- Installed babel modules
- Created script `npm run sequelize *` to transpile database files to es5 before running.
[Finishes #164198481]

danprocoder committed:
chore(databaseFiles): Refactored database files to ES6 syntaxRefactored files in /database to follow ES6 syntax[Finishes #164198481]

danprocoder committed:
chore(packageLock): Fixed merge conflict
[Finishes #164198481]

danprocoder committed:
chore(package, modelIndex): Fixed merge conflict and updated models/index.js
- Fixed merge conflict in package.json
- Changed `module.exports` to `export default` in database/models/index.js
[Finishes #164198481]
